modules/ROOT/pages/runtime-manager-agent.adoc

```diff
@@ -7,70 +7,74 @@ endif::[]
+* Your enterprise license is current.
++
+For information on obtaining and installing an enterprise license, see xref:4.1@mule-runtime::installing-an-enterprise-license.adoc[Installing an Enterprise License].
 
-When registering the Runtime Manager agent with Runtime Manager (by using the hybrid `-H` option), the Runtime Manager agent first makes an outbound HTTPS connection to the Runtime Manager. This connection is used to establish a new secure, full duplex WebSocket connection between the Runtime Manager and the Runtime Manager agent.
+== Installation Location
 
-Once configured, the Runtime Manager agent securely connects to this Runtime Manager account every time the Mule restarts.  The link that the Runtime Manager agent establishes allows the Mule to be monitored and managed remotely through an Anypoint Runtime Management console.
+Runtime Manager agent is installed into your `$MULE_HOME/plugins` directory and reads its configuration from a single `mule-agent.yml` file. For installation instructions and download links, see xref:installing-and-configuring-runtime-manager-agent.adoc[Install or Update Runtime Manager Agent].
 
-If this link goes down while the Mule is still running, statistics and analyics are buffered until the WebSocket connection can be re-established. Additionally, there is also a reconnection strategy in place in cases the WebSocket connection drops.
 
-=== REST Communication
-You can configure each Runtime Manager agent to allow other REST connections (either secure with the `-S` option, or insecure with the `-I` option), for use by other management tools outside of Runtime Manager.
+== Communication Protocols
 
-You can also manually edit the Runtime Manager agent's `mule-agent.yml` file to combine communication connections into one Runtime Manager agent, and also to modify parameters of these connections.
+Runtime Manager agent supports two communication protocols: 
 
-== Installation Location
-The Runtime Manager agent is installed into your `$MULE_HOME/plugins` directory and is configured via a single configuration file. For installation instructions and download links, see xref:installing-and-configuring-runtime-manager-agent.adoc[installing and configuring Runtime Manager agent].
+* *WebSockets*
++ 
+Establishes a connection from the agent to Runtime Manager (cloud-based or PCE console).
++
+When you register the agent with Runtime Manager, the agent first makes an outbound HTTPS connection to Runtime Manager. This connection establishes a new secure, full-duplex WebSocket connection between Runtime Manager and the agent. When Mule starts, the agent securely connects to Runtime Manager, enabling you to monitor and manage Mule remotely using the Runtime Manager console.
++
+If this link goes down while Mule is still running, statistics and analytics are buffered until the WebSocket connection is reestablished. Additionally, there is also a reconnection strategy in place in cases the WebSocket connection drops.
```

Comment:
```suggestion
If this link goes down while Mule is still running, statistics and analytics are buffered until the WebSocket connection is reestablished. Additionally, a reconnection strategy is in place in case the WebSocket connection drops.
```